### ISSUE TITLE
Test fields `actual_time` and `estimated_time`

### ISSUE THREAD

**Issue by fopina** [DefectDojo/django-DefectDojo#11121]
https://github.com/DefectDojo/django-DefectDojo/blob/5c7de81ee8405df6b1a6844b912aaf3bd7d3d2a1/dojo/models.py#L2038

I just spent about an hour trying to set `actual_time` when update a test via REST API and then looking through the code to see how if it was used anywhere...

To be honest, documentation doesn't mention it as possible request payload anywhere, but it's part of the output. I don't see it in the UI either.

Is it something that was deprecated? I thought it could be a good place to store scanner run times for metrics later, just checking as I can use any other field

**Comment by mtesauro** [comment]
@fopina If memory serves, the actual_time field was added to DefectDojo to handle the case when manually testing something where there's an estimated time (e.g. 5 days) but the time to complete that manual testing ends up being different (e.g. 4 days).

That allows you to track the accuracy of your estimates vs reality when managing manual testing efforts.

It's not deprecated but likely only created in the UI since it was designed to be used with manual testing efforts.

**Comment by fopina** [comment]
Right, that makes sense however I couldn’t find it anywhere in the UI either 🤷 
I’ll see if I find something in history about it. If it wasn’t removed, just never used, I guess it might be acceptable doing a PR to expose it

**Comment by valentijnscholten** [comment]
@mtesauro This looks like a left over field not exposed / used anywhere. Shall we remove it?

**Comment by mtesauro** [comment]
Yes, go for it (which you've already done 😉 )
